fplll/nr/nr_Z_mpz.inl

```diff
@@ -11,7 +11,7 @@ FPLLL_BEGIN_NAMESPACE
 template <> inline Z_NR<mpz_t>::Z_NR() { mpz_init(data); }
 
 template <> inline Z_NR<mpz_t>::Z_NR(const Z_NR<mpz_t> &z) { mpz_init_set(data, z.data); }
-
+template <> inline Z_NR<mpz_t>::Z_NR(const mpz_t &z) { mpz_init_set(data, z); }
```

Comment:
I think this needs documentation that this does not take ownership of z, i.e.the caller will have to still `mpz_free` it after.

Reply:
Great catch - I've added it. Adding move constructors in future would be useful, but doing this (and testing it doesn't implicitly break anything) is probably an Fplll days task...

Reply:
You added it to `operator=` but shouldn't we add it to both?

Reply:
I've agree - it's been added.